### PR TITLE
CLOUD-60536 New catatnight/postfix container for email sending

### PR DESCRIPTION
**Pull request by schfeca75**
@akanto or @lalyos pls review

### CODE REVIEW

**Comment by lalyos** [comment]
lgtm,
please make sure, you add a couple of line of documentation

File: include/compose.bash [hunk 1]
```diff
@@ -261,6 +261,15 @@ logspout:
     entrypoint: ["/bin/sh"]
     command: -c 'sleep 1; ROUTE_URIS=\$\$LOGSINK_PORT_3333_TCP /bin/logspout'
 
+mail:
+    ports:
+        - "$PRIVATE_IP:25:25"
+    environment:
+        - maildomain=mail.hortonworks.com
+        - smtp_user=admin:cloudbreak
```
Comment (mhmxs) [inline include/compose.bash]: This looks a username password pair for me. Maybe environment variables would be better .

File: include/cloudbreak.bash [hunk 2]
```diff
@@ -27,6 +27,7 @@ cloudbreak-conf-tags() {
     env-import DOCKER_TAG_CONSUL 0.5
     env-import DOCKER_TAG_REGISTRATOR v5
     env-import DOCKER_TAG_POSTGRES 9.4.1
+    env-import DOCKER_TAG_POSTFIX latest
```
Comment (mhmxs) [inline include/cloudbreak.bash]: Please fixme, but we usually try to avoid latest builds.

Comment (schfeca75) [inline include/cloudbreak.bash]: It has only latest version

Comment (mhmxs) [inline include/cloudbreak.bash]: On that way we should fork project as hortonworks and create our own build.

Comment (lalyos) [inline include/cloudbreak.bash]: No, please stop forking repositories. A better solution is to refer by image ID.

File: include/compose.bash (hunk 1 above)
Comment (schfeca75) [inline include/compose.bash]: If you would like to use this as smtp server, you have to set the following env vars:
export CLOUDBREAK_SMTP_SENDER_USERNAME=admin
export CLOUDBREAK_SMTP_SENDER_PASSWORD=cloudbreak
...

Comment (mhmxs) [inline include/compose.bash]: But admin:cloudbreak is a hard coded value, not a variable!?

**Pull request merged**